src/AWS/Lambda/Events/EventBridge/SSM/ParameterStoreChange.hs

```diff
@@ -0,0 +1,132 @@
+{-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+-- |
+-- Module      : AWS.Lambda.Events.EventBridge.SSM.ParameterStoreChange
+-- Description : Data types for AWS Systems Manager Parameter Store Change events.
+-- License     : BSD3
+-- Stability   : stable
+module AWS.Lambda.Events.EventBridge.SSM.ParameterStoreChange
```

Comment:
Since it's the only thing which varies within an `EventBridgeEvent` I felt this was okay, but I'm happy to tweak that if you'd like. I'm not aware of them being used for anything else - I can't even find them on the EventBridge schema registry! The only place that I could find was a couple of pages in the [Systems Manager User Guide](https://docs.aws.amazon.com/systems-manager/latest/userguide/monitoring-systems-manager-event-examples.html#SSM-Parameter-Store-event-types)

So this is all a bit stab-in-the-dark. I'd be OK with `AWS.Lambda.Events.EventBridge.Detail.SSM.ParameterStoreChage`, if you felt like you wanted more room.

Reply:
> Since it's the only thing which varies within an EventBridgeEvent I felt this was okay

Yeah, that does make sense.  `Detail` over `DetailTypes` does seem better in that it only adds seven characters, but it's just so long already!  No perfect answer here; I'm happy to merge as is.

Reply:
I've renamed to Detail, in case we need to flex more on other fields.